wavefront/templates/proxy-deployment.yaml

```diff
@@ -61,6 +62,10 @@ spec:
         - containerPort: {{ .Values.proxy.jaegerPort }}
           protocol: TCP
         {{- end }}
+        {{- if .Values.proxy.traceJaegerHttpListenerPorts }}
+        - containerPort: {{ .Values.proxy.traceJaegerHttpListenerPorts }}
```

Comment:
To add to the other comment, the container port is a singular port. So let's support setting only one port for this at this time.